wrappers/python/examples/auto_calibration_config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The content should be documented.
Also, the file seem to belong to infrastructure, consider to move it to root/common or config